### PR TITLE
Fix for ZeroDivisionError: float division by zero in pagerank_weighted.py #851

### PR DESCRIPTION
**Pull request by metalaman**
Returns the original text as soon as it finds only one sentence present in the document.

### CODE REVIEW

**Comment by tmylk** [comment]
Please also add a test and a line in CHANGELOG

**Comment by tmylk** [comment]
Please run tests locally before committing to the main repo

**Comment by metalaman** [comment]
For some reason, this build terminated for Python 2.6, ran successfully for all other version.

**Comment by metalaman** [comment]
So, I removed the logging statement and it passed. I don't understand how.

File: gensim/test/test_summarization.py [hunk 1]
```diff
@@ -88,6 +88,17 @@ def test_text_summarization_raises_exception_on_short_input_text(self):
...
+    def test_text_summarization_returns_input_on_single_input_sentence(self):
+        pre_path = os.path.join(os.path.dirname(__file__), 'test_data')
+
+        with utils.smart_open(os.path.join(pre_path, "testsummarization_unrelated.txt"), mode="r") as f:
+            text = f.read()
+
+        # Keeps the first sentence only.
+        text = text.split('\n')[0]
+
+        self.assertTrue(summarize(text) is not None)
```
Comment (tmylk) [inline gensim/test/test_summarization.py]: Please assert that the return is equal to text.

**Comment by tmylk** [comment]
@metalaman  Thanks a lot for the PR!

**Pull request merged**